hamilflow/models/brownian_motion.py

```diff
@@ -0,0 +1,205 @@
+from functools import cached_property
+from typing import Dict, List, Optional, Union
+
+import numpy as np
+import pandas as pd
+import scipy as sp
```

Comment:
`scipy` is very comprehensive. We can just `from scipy.stats import norm` to be more explicit.

Reply:
I intentionally choose to use the top name space for such short names. You probably noticed I also do everything for np and pd. So here I intent to keep the coding style consistent.

There is not actual difference in performance and I think using `sp` makes is easier for me to read and reduce cognitive load. Let's do an example:

1. `norm`: I will have to think about where I got `norm`.
2. `sp.stats.norm`: ah the norm is from scipy not numpy or pandas.